webroot/index.php

```diff
@@ -24,14 +24,14 @@
         return false;
     }
 }
-require dirname(__DIR__) . '/config/bootstrap.php';
+require dirname(__DIR__) . '/vendor/autoload.php';
```

Comment:
Hm... People could wonder where or even if bootstrap is now called.

May be add a note down where ``Application`` gets instantiated?